src/Index.ts

```diff
@@ -86,30 +86,47 @@ app.use('/api/v1', isAuthenticated, (req, res, next) => {
 
 let serverHttps: any
 if (config.https) {
-  const WebSocketCertificatePath = path.join(__dirname, EnvReader.GlobalEnvConfig.WSConfiguration.WebSocketCertificate)
-  const WebSocketCertificateKeyPath = path.join(__dirname, EnvReader.GlobalEnvConfig.WSConfiguration.WebSocketCertificateKey)
-  let RootCACertPath
-  if (EnvReader.GlobalEnvConfig.WSConfiguration.RootCACert) {
-    RootCACertPath = path.join(__dirname, EnvReader.GlobalEnvConfig.WSConfiguration.RootCACert)
-    if (!existsSync(RootCACertPath)) {
-      log.error(`Root cert ${RootCACertPath} doesn't exist. Exiting..`)
+  let certs: any
+
+  if (EnvReader.GlobalEnvConfig.DbConfig.useRawCerts) {
+    log.debug('using raw certs')
+
+    certs = {
+      webConfig: {
+        key: EnvReader.GlobalEnvConfig.WSConfiguration.WebSocketCertificateKey,
```

Comment:
Lets restructure this a bit such that the `EnvReader.GlobalEnvConfig.WSConfiguration.X` are read once into variables, and then the appropriate action is taken based on the useRawCerts. The `certs = ` should only need to be set once.